package.json

```diff
@@ -1,23 +1,25 @@
 {
   "name": "nuxt-icon",
+  "packageManager": "pnpm@9.0.5",
   "version": "0.6.10",
   "license": "MIT",
   "type": "module",
   "repository": "nuxt-modules/icon",
   "exports": {
     ".": {
-      "types": "./dist/types.d.ts",
+      "types": "./dist/module.d.ts",
```

Comment:
@antfu Why not use `./dist/types.d.ts` anymore?

Reply:
I am not sure, but the latest module-builder no longer generates that file for some reason.